wiki/Modding/en.md

```diff
@@ -1,98 +1,111 @@
 
-Next, go into the [Editor](Editor "wikilink") and select the map which you want to mod. Then, start up a reply to the [Beatmap's](Beatmap "wikilink") thread.
+Next, go into the [editor](/wiki/Beatmap_Editor) and select the map which you want to mod. Then, start up a reply to the [beatmap's](/wiki/Beatmaps) thread.
 
 Choose any of the difficulties to begin your mod. It is recommended to start with the easiest or hardest difficulty and make your way up or down the different difficulties.
 
-Start by playing this difficulty in either Play mode or [test mode](Glossary#Test_Mode "wikilink"). Play-testing is an important part of modding as it allows the modder to understand whether or not certain [patterns](Mapping_Techniques "wikilink") play well. Take note of anything that felt odd.
+Start by playing this difficulty in either play mode or [test mode](/wiki/Glossary/#test-mode). Play-testing is an important part of modding as it allows the modder to understand whether or not certain [patterns](/wiki/Mapping_Techniques) play well. Take note of anything that felt odd.
 
-After play-testing, watch the song in the editor and pause frequently to catch any errors. In this stage of modding, it is most often that modders find more intricate problems such as combo errors, hitsound inconsistencies, and areas where hit circles or sliders are missing. When you find a hit circle or a slider that needs attention, select it. Now, go to the "Edit" tab and select "Copy" at the top (or Ctrl+C), and then to your post you started up, and paste (Ctrl+V) what you just copied there. Then, follow up with either what you think is wrong (Ex. "I would change this spacing because the music does not suggest a [jump](jumps "wikilink") here), or simply that it doesn't seem right and you can't put your finger on it. Always look for things that you didn't notice while playing.
+After play-testing, watch the song in the editor and pause frequently to catch any errors. In this stage of modding, it is most often that modders find more intricate problems such as combo errors, hitsound inconsistencies, and areas where hit circles or sliders are missing. When you find a hit circle or a slider that needs attention, select it. Now, go to the `Edit` tab and select `Copy` at the top (or `Ctrl` + `C`), and then to your post you started up, and paste (`Ctrl` + `V`) what you just copied there. Then, follow up with either what you think is wrong (Ex. "I would change this spacing because the music does not suggest a [jump](/wiki/Mapping_Techniques/Jumps) here), or simply that it doesn't seem right and you can't put your finger on it. Always look for things that you didn't notice while playing.
```

Comment:
```Markdown
After play-testing, watch the song in the editor and pause frequently to catch any errors.
In this stage of modding, it is most often that modders find more intricate problems such as combo errors, hitsound inconsistencies, and areas where hit circles or sliders are missing.
When a hit circle or a slider that needs attention was found, select it.
Now, go to the `Edit` tab and select `Copy` at the top (or `Ctrl` + `C`), and then to your post you started up, and paste (`Ctrl` + `V`) what you just copied there.
Then, follow up with either what you think is wrong (Ex. "I would change this spacing because the music does not suggest a [jump](/wiki/Mapping_Techniques/Jumps) here), or simply that it does not seem right and you cannot put your finger on it.
Always look for things that you did not notice while playing.
```

Need double-checking. I am stuck at the moment.